docs/integrations.md

```diff
@@ -40,6 +40,46 @@ The Go template can utilize helper functions from [Sprig].
 
 For example templates, see [tracee/cmd/tracee-rules/templates].
 
+## Container Enrichment
+
+Tracee is capable of extracting information on running containers on your system during runtime through tracking created cgroups in kernel and can further enrich the container events from data queried by communicating with the relevant container's runtime and SDK from userspace.
+
+Since this feature is still experimental, it is disabled by default and must be explicitely enabled.
+
+In order to enable this capability in the container's trace mode or with tracee-ebpf's binary, use the `--containers` flag.
+If running tracee with tracee-rules, set the environment flag `CONTAINERS_ENRICHMENT` (example below).
+
+If running tracer-ebpf directly from binary, it will automatically search for known supported runtimes in their default socket's locations.
```

Comment:
```suggestion
If running tracee-ebpf directly (not in a container), it will automatically search for known supported runtimes in their socket's default locations.
```